tools/topology/topology1/sof-acp-rmb-tdm8ch-dmic2ch-nuvoton.m4

```diff
@@ -0,0 +1,178 @@
+#
+# Topology for Renoir with I2S SP and DMIC.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`acp-hs.m4')
+include(`acp-dmic.m4')
+include(`m4/debug.m4')
+include(`muxdemux.m4')
+include(`bytecontrol.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include ACP DSP configuration
+include(`platform/amd/acp.m4')
+
+
+DEBUG_START
+
+define(PIPE_NAME, pipe-acp-mux)
+
+#
+# Define the pipelines
+#
+# PCM0 ----> volume ---------------+
+#                                  |--low latency mixer ----> volume ---->  ACPSP_play
+# PCM1 -----> volume ----->     ---+
+#
+# PCM0 <---- Volume <---- ACPSP_Capture
+#
+
+#/**********************************************************************************/
+mux pipe1 and pipe3
+define(matrix1, `ROUTE_MATRIX(1,
+			     `BITS_TO_BYTE(0, 0, 0 ,0 ,1 ,0 ,0 ,0)',
+			     `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,1 ,0 ,0)',
+			     `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,1 ,0)',
+			     `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,0 ,1)',
+			     `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,0 ,0)',
+			     `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,0 ,0)',
+			     `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,0 ,0)',
+			     `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,0 ,0)')')
+
+define(matrix2, `ROUTE_MATRIX(3,
+			     `BITS_TO_BYTE(0, 0, 1 ,0 ,0 ,0 ,0 ,0)',
+			     `BITS_TO_BYTE(0, 0, 0 ,1 ,0 ,0 ,0 ,0)',
+			     `BITS_TO_BYTE(1, 0, 0 ,0 ,0 ,0 ,0 ,0)',
+			     `BITS_TO_BYTE(0, 1, 0 ,0 ,0 ,0 ,0 ,0)',
+			     `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,0 ,0)',
+			     `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,0 ,0)',
+			     `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,0 ,0)',
+			     `BITS_TO_BYTE(0, 0, 0 ,0 ,0 ,0 ,0 ,0)')')
```

Comment:
Mixer or mux? This is very confusing, the description says mixer and you have parameters for mux. Which is it?

Reply:
updated header, its mux.